ea_hyperlink_label.m

```diff
@@ -1,4 +1,20 @@
 function ea_hyperlink_label(label, url, position)
+% 
```

Comment:
Please provide a description of the function

Reply:
```
%  Create a clickable hyperlink label
%
% INPUTS:
%    label: label text
%    url: linked URL
%    position: position of the label
```